src/scale-manager/README.md

```diff
@@ -54,7 +54,7 @@ For more info check out the [WebKit Bug](https://bugs.webkit.org/show_bug.cgi?id
 
 ### Callback Example
 
-The safe scale manager's callback parameter contains `width`, `height`, and `scale`. these values can be used to resize your game. Depending on the engine you choose you'll have to use them differently. `scale` contains `x` and `y`, these are the independent scaling values for the x and y axis.
+The safe scale manager's callback parameter contains `width`, `height`, `scale`, and `scaleRatio`. these values can be used to resize your game. Depending on the engine you choose you'll have to use them differently. `scale` contains `x` and `y`, these are the independent scaling values for the x and y axis. You may also need to use the `scaleRatio` value, this is the minimum size ratio either `width/safeWidth` or `height/safeHeight`
```

Comment:
Let's also document that `scaleRatio` is also available outside of resize events if needed.

Reply:
I added a table to the safe scale manager readme that outlines "public" member variables. This adds clarity to what each value is used for within the manager and calls out that `scaleRatio` and `viewArea` are also accessible outside of the resize event.